### PR TITLE
Theme/multisite 13187

### PR DESCRIPTION
**Pull request by MehdiDonk**

### CODE REVIEW

File: lib/themes/erasmus/template.php [hunk 1]
```diff
@@ -157,6 +157,7 @@ function erasmus_preprocess_om_maximenu_submenu(&$variables) {
  */
 function erasmus_preprocess_page(&$variables, $hook) {
   $header = drupal_get_http_header("status");
+
```
Comment (jonhy81) [inline lib/themes/erasmus/template.php]: No withspace need, only a white line. Please fix here and in the rest of the file. 
 160 | ERROR | [x] Whitespace found at end of line
 251 | ERROR | [x] Whitespace found at end of line
 253 | ERROR | [x] Whitespace found at end of line
 256 | ERROR | [x] Whitespace found at end of line
 261 | ERROR | [x] Whitespace found at end of line
 265 | ERROR | [x] Whitespace found at end of line
 273 | ERROR | [x] Whitespace found at end of line
 285 | ERROR | [x] Whitespace found at end of line
 289 | ERROR | [x] Whitespace found at end of line
 293 | ERROR | [x] Whitespace found at end of line

Comment (jonhy81) [inline lib/themes/erasmus/template.php]: Remove whitspaces, only empty line should be added. 
 160 | ERROR | [x] Whitespace found at end of line
 251 | ERROR | [x] Whitespace found at end of line
 253 | ERROR | [x] Whitespace found at end of line
 256 | ERROR | [x] Whitespace found at end of line
 261 | ERROR | [x] Whitespace found at end of line
 265 | ERROR | [x] Whitespace found at end of line
 273 | ERROR | [x] Whitespace found at end of line
 285 | ERROR | [x] Whitespace found at end of line
 289 | ERROR | [x] Whitespace found at end of line
 293 | ERROR | [x] Whitespace found at end of line

File: lib/themes/erasmus/template.php [hunk 2]
```diff
@@ -181,55 +182,55 @@ function erasmus_preprocess_page(&$variables, $hook) {
...
-        action=index&serviceid=1756&pk_campaign=SubscribeNewsletter&pk_kwd=Modal",
-        array(
-          'attributes' => array(
-            'class' => 'link-modal',
-            'align' => 'center',
-          ),
-        )
-      );
+          t('Subscribe to our newsletter'),
+          "https://ec.europa.eu/coreservices/mailing/index.cfm?controller=register&
```
Comment (jonhy81) [inline lib/themes/erasmus/template.php]: Move the URL query to the 3rd argument of the l() call.

Comment (jonhy81) [inline lib/themes/erasmus/template.php]: Also check the indentation, in L185 by example you have 12 spaces and should be 4. Check also all new code.

File: lib/themes/erasmus/template.php [hunk 3]
```diff
@@ -247,54 +248,52 @@ function erasmus_preprocess_page(&$variables, $hook) {
     $variables['colheightortwelve'] = 'col-lg-12';
     $variables['bgpage'] = 'frontbg';
   }
-
+
```
Comment (jonhy81) [inline lib/themes/erasmus/template.php]: Leave only a empty line without with spaces

Comment (MehdiDonk) [inline lib/themes/erasmus/template.php]: Done

File: lib/themes/erasmus/template.php (hunk 2 above)
Comment (MehdiDonk) [inline lib/themes/erasmus/template.php]: I reviewed the indentation of the whole page.
I am not sure to get what you mean by
"Move the URL query to the 3rd argument of the l() call."

File: lib/themes/erasmus/template.php (hunk 1 above)
Comment (MehdiDonk) [inline lib/themes/erasmus/template.php]: Done

File: lib/themes/erasmus/template.php [hunk 4]
```diff
@@ -44,112 +44,133 @@ function erasmus_preprocess_om_maximenu_submenu(&$variables) {
...
-            'data-expand' => 'mobile-nav-newsletter',
-          ),
-        )
-    );
+    $empty, "https://plus.google.com/communities/115077875344973792963",
+    array('attributes' => array('class' => 'icon gplus'))
+  );
+  $title = $empty;
+  $url = 'https://ec.europa.eu/coreservices/mailing/index.cfm';
+  $options =   array(
```
Comment (jonhy81) [inline lib/themes/erasmus/template.php]: It is almost green. please remove the additional spaces after the operator in the folowings lines. 
 60 | ERROR | [x] Expected 1 space after "="; 3 found
 77 | ERROR | [x] Expected 1 space after "="; 3 found
 94 | ERROR | [x] Expected 1 space after "="; 3 found

Comment (MehdiDonk) [inline lib/themes/erasmus/template.php]: Done

**Pull request merged**